pom.xml

```diff
@@ -77,4 +77,31 @@ THE SOFTWARE.
         </pluginRepository>
     </pluginRepositories>
 
+    <profiles>
+        <profile>
+            <id>jacoco</id>
+            <build>
+                <plugins>
+                    <plugin>
+                        <groupId>org.jacoco</groupId>
+                        <artifactId>jacoco-maven-plugin</artifactId>
+                        <version>0.8.5</version>
+                        <executions>
+                            <execution>
+                                <goals>
+                                    <goal>prepare-agent</goal>
+                                </goals>
+                            </execution>
+                        </executions>
+                        <configuration>
+                            <excludes>
+                                <exclude>**/Messages.class</exclude>
```

Comment:
Curious why exclude that class?

Reply:
I found casc using it, so I thought it must be some by-productish class that gets generated as a result of a run, since they do not have any such class in their repo.

However, Oleg suggested that we instead use ci.jenkins.io for this by configuring the Jacoco plugin. What would be your thoughts? :)